tests/conftest.py

```diff
@@ -16,17 +17,27 @@ def is_responsive(url):
         return False
 
 
+@pytest.fixture(scope="session")
+def screenshot_dir():
+    sdir = Path.joinpath(Path.cwd(), "screenshots")
+    try:
+        os.mkdir(sdir)
+    except FileExistsError:
+        pass
+    return sdir
+
+
 @pytest.fixture(scope="session")
 def notebook_service(docker_ip, docker_services):
     """Ensure that HTTP service is up and responsive."""
 
     docker_compose = docker_services._docker_compose
 
     # assurance for host user UID other that 1000
-    chown_command = "exec -T -u root aiidalab bash -c 'chown -R jovyan:users /home/jovyan/apps/aiidalab-widgets-base'"
+    chown_command = "exec -T -u root aiidalab bash -c 'chmod -R a+rw /home/jovyan/apps/aiidalab-widgets-base'"
     docker_compose.execute(chown_command)
 
-    install_command = "bash -c 'pip install -U .'"
+    install_command = "bash -c 'pip install .'"
```

Comment:
As I comment on https://github.com/aiidalab/aiidalab-widgets-base/pull/409, I think here we want to keep all dependencies updated, no?

Hmmm.... I think I understand your point, you are worry that the `-U` will conflict and override other dependencies? 
I think we need other tests to test the conflicting dependencies. Here we want to test the dependencies we set here is working.

Reply:
The dependencies should be satisfied even without the -U command. Since we're typically installing into a new Python environment, the -U shouldn't really have any effect. I am not quite sure how it interacts with pip cache, but we bust the cache anyway on any change to setup.cfg so it shouldn't matter.